src/app/modules/form/form.utils.ts

```diff
@@ -100,6 +101,18 @@ export const getCollabEmailsWithPermission = (
   }, [])
 }
 
+export const getAdminEmails = (
+  form: Partial<IFormHasEmailSchema>,
+): string[] => {
+  if (form.emails === undefined) {
```

Comment:
Would prefer if we check for null + undefined with `form.emails == null`

Reply:
Oh wow, didn't know that the weaker equality `==` accounted for `undefined` and `null` as well. Resolved, thanks!